tests/testthat/test-build-home-index.R

```diff
@@ -69,3 +69,9 @@ test_that("homepage description can be overridden", {
 
 })
 
+test_that("header attributes are parsed", {
```

Comment:
Can you please move this test to `test-markdown.R` since it's testing `markdown()`? Thanks!

Reply:
Done!